### ISSUE TITLE
Besu spamming the logs with pandas pre-merge

### ISSUE THREAD

**Issue by ajsutton** [hyperledger/besu#3843]
### Description
Spun up a new besu node on ropsten running version `besu/v22.4.1-dev-3835cb72/linux-x86_64/corretto-java-17`.

Sync hasn't even completed and the logs are continually being spammed with pandas:

```
2022-05-17 00:40:19.516+00:00 | main | INFO  | DefaultP2PNetwork | Enode URL enode://1c03b5b94d0fcb0a99f7c07bfd8982e199eb4ecd00c8e60d25ef6ef87e1fb32a4482da3f1b208cbd88e614dca678d68f5f509e6ee7d685148c2d25b7bb6db3e0@18.117.226.181:30303
2022-05-17 00:40:19.516+00:00 | main | INFO  | DefaultP2PNetwork | Node address 0x8370b329af7145e2ac2a285ddf7c5bb906888913
2022-05-17 00:40:19.521+00:00 | main | INFO  | DefaultSynchronizer | Starting synchronizer.
2022-05-17 00:40:19.522+00:00 | main | INFO  | FastSyncDownloader | Starting fast sync.
2022-05-17 00:40:19.527+00:00 | main | INFO  | WaitForPeersTask | Waiting for 5 total peers to connect. 0 peers currently connected.
2022-05-17 00:40:19.530+00:00 | main | INFO  | Runner | Ethereum main loop is up.
2022-05-17 00:40:19.791+00:00 | nioEventLoopGroup-3-2 | INFO  | WaitForPeersTask | Waiting for 5 total peers to connect. 1 peers currently connected.
2022-05-17 00:40:22.229+00:00 | Timer-0 | INFO  | DNSResolver | Resolved 426 nodes
2022-05-17 00:40:23.400+00:00 | nioEventLoopGroup-3-9 | INFO  | WaitForPeersTask | Waiting for 5 total peers to connect. 2 peers currently connected.
2022-05-17 00:40:23.776+00:00 | nioEventLoopGroup-3-9 | INFO  | WaitForPeersTask | Complete: 5 peers connected.
2022-05-17 00:40:23.778+00:00 | nioEventLoopGroup-3-9 | INFO  | FastSyncActions | Waiting for valid peers with chain height information.  1 / 5 required peers currently available.
2022-05-17 00:40:28.780+00:00 | EthScheduler-Timer-0 | INFO  | PivotSelectorFromPeers | Selecting block number 12270938 as fast sync pivot block.
2022-05-17 00:40:28.782+00:00 | EthScheduler-Timer-0 | INFO  | PivotBlockRetriever | Retrieve a pivot block that can be confirmed by at least 5 peers.
2022-05-17 00:40:28.784+00:00 | EthScheduler-Timer-0 | INFO  | PivotBlockConfirmer | Confirm pivot block 12270938 with at least 5 peers.
2022-05-17 00:40:28.880+00:00 | nioEventLoopGroup-3-4 | INFO  | PivotBlockConfirmer | Received 1 confirmation(s) for pivot block header 12270938: 0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6
2022-05-17 00:40:28.880+00:00 | nioEventLoopGroup-3-6 | INFO  | PivotBlockConfirmer | Received 2 confirmation(s) for pivot block header 12270938: 0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6
2022-05-17 00:40:28.882+00:00 | nioEventLoopGroup-3-9 | INFO  | PivotBlockConfirmer | Received 3 confirmation(s) for pivot block header 12270938: 0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6
2022-05-17 00:40:28.883+00:00 | nioEventLoopGroup-3-10 | INFO  | PivotBlockConfirmer | Received 4 confirmation(s) for pivot block header 12270938: 0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6
2022-05-17 00:40:33.897+00:00 | nioEventLoopGroup-3-9 | INFO  | PivotBlockConfirmer | Confirmed pivot block at 12270938: 0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6
2022-05-17 00:40:33.901+00:00 | nioEventLoopGroup-3-9 | INFO  | FastWorldStateDownloader | Begin downloading world state from peers for block 12270938 (0x50653371839c7a1d2a99431bc34e087cde50ae3419e3fc73fd9221713502d9f6). State root 0x9077
838d8397744d16b53571240293a29923a7e1afd913797468abc9d9981ccb
2022-05-17 00:40:34.016+00:00 | nioEventLoopGroup-3-4 | INFO  | SyncTargetManager | Found common ancestor with peer Peer 0x1b114022f2e6cfcc89... PeerReputation 100, validated? true, disconnected? false at block 0
2022-05-17 00:40:35.574+00:00 | EthScheduler-Services-13 (importBlock) | INFO  | PandaPrinter |
<snip> panda art
2022-05-17 00:40:35.580+00:00 | EthScheduler-Services-13 (importBlock) | INFO  | PandaPrinter |
<snip> panda art
2022-05-17 00:40:35.581+00:00 | EthScheduler-Services-13 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
2022-05-17 00:40:35.582+00:00 | EthScheduler-Services-13 (importBlock) | INFO  | PandaPrinter |
<snip> panda art
```

The panda art should only print when the merge is complete (beacon chain hasn't even started in this case) and definitely should only ever print once.

**Comment by macfarla** [comment]
Should be fixed by https://github.com/hyperledger/besu/pull/3844

**Comment by ajsutton** [comment]
#3844 stops it repeatedly printing pandas but it's still printing once at startup well before the merge has happened.

**Comment by macfarla** [comment]
From Adrian: 
Also these logs from ropsten
```
2022-05-17 02:46:07.891+00:00 | EthScheduler-Services-34 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
2022-05-17 02:46:07.910+00:00 | EthScheduler-Services-34 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
2022-05-17 02:46:07.930+00:00 | EthScheduler-Services-34 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
2022-05-17 02:46:07.937+00:00 | EthScheduler-Services-34 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
2022-05-17 02:46:07.939+00:00 | EthScheduler-Services-34 (importBlock) | WARN  | BlockPropagationManager | Attempted to stop when we are not even running...
```
Teku is telling me that besu has TTD set to 0x0000000000000000000000000000000000000000000000000000000000000000 which I’d expect if the ropsten config hasn’t been updated yet but not sure if it’s really set to 0 and that’s why it thinks it has been reached.

**Comment by matt-nelson-csi** [comment]
Is this considered fixed via #3844? If we are awaiting Ropsten config, can consider this closed.
